resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -9,6 +9,7 @@ loading_resource.failed=Failed loading resources
 explorer.label=AWS Explorer
 explorer.view_documentation=View Documentation
 explorer.view_source=View Source on GitHub
+explorer.cut_issue=Cut an Issue on GitHub
```

Comment:
I think cut is more of an amazon term, I would use "Create"

Reply:
Can we do "Create a new issue on Github" since the button to get there is called "New Issue"

Reply:
Done. Still using the title case "Create a New Issue on GitHub" to be consistent with other action titles.